edu.stanford.protege.json

```diff
@@ -0,0 +1,74 @@
+{
+    "id": "edu.stanford.protege",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk8"],
+    "command": "protege",
+    "finish-args": [
+        "--env=PATH=/app/bin:/app/jre/bin:/usr/bin",
+        "--socket=x11",
+        "--share=ipc",
+        "--share=network",
+        "--filesystem=home"
```

Comment:
that's a wide filesystem permission

Reply:
It's kind of common to open OWL files from downloads or the desktop in a typical workflow.

Reply:
There are standard xdg locations, like `xdg-desktop`, `xdg-documents` and, `xdg-download`